packages/core/src/browser/style/tree.css

```diff
@@ -140,3 +140,7 @@
 .theia-tree-element-node {
     width: 100%
 }
+
+.theia-TreeNodeSubLabel:active{
+    color: var(--theia-list-activeSelectionForeground) !important;
```

Comment:
I don't think it is the same color in VS Code for primary and description foregrounds in active mode

<img width="467" alt="Screenshot 2020-02-25 at 11 24 13" src="https://user-images.githubusercontent.com/3082655/75238513-6a2f2280-57c1-11ea-9284-b1f38481da98.png">